src/login/style/hooks/use-mobile-style.js

```diff
@@ -0,0 +1,11 @@
+import { makeStyles } from '@material-ui/core'
+
+const root = {
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/126044433-ca9cca78-39f2-41a3-8dc5-b2eeac1ce757.png)
These two styles are really similar. Can we extract these common attributes in a separate style or object?

Reply:
Done